app/Filament/Components/Actions/ImportEggAction.php

```diff
@@ -44,27 +45,39 @@ protected function setUp(): void
                     Tab::make(trans('admin/egg.import.url'))
                         ->icon('tabler-world-upload')
                         ->schema([
-                            TextInput::make('url')
-                                ->default(fn (Egg $egg) => $egg->update_url)
-                                ->label(trans('admin/egg.import.url'))
+                            Repeater::make('url')
+                                ->label('Egg')
                                 ->hint(trans('admin/egg.import.url_help'))
-                                ->url(),
+                                ->addActionLabel(trans('admin/egg.import.add_url'))
+                                ->unique()
+                                ->reorderable(false)
+                                ->schema([
```

Comment:
Make the repeater [simple](https://filamentphp.com/docs/3.x/forms/fields/repeater#simple-repeaters-with-one-field)